From 8fca2a8eed6ebc1cbf9f7972c6a9bb137ebafe1a Mon Sep 17 00:00:00 2001
From: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>
Date: Sun, 12 Dec 2021 17:07:03 +0000
Subject: [PATCH] Increase SMM gateway UEFI variable macro value

The maximum number of UEFI variables that be supported by SMM
gateway is currently 40. When more than 40 variables are written,
or read SMM gateway returns error code. Currently this value is
increased to 100 to support more UEFI variables.

Signed-off-by: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>

Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>


---
 deployments/smm-gateway/smm_gateway.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/deployments/smm-gateway/smm_gateway.c b/deployments/smm-gateway/smm_gateway.c
index 7828b3af..7e6729ee 100644
--- a/deployments/smm-gateway/smm_gateway.c
+++ b/deployments/smm-gateway/smm_gateway.c
@@ -20,6 +20,9 @@
 #define SMM_GATEWAY_NV_STORE_SN		"sn:ffa:751bf801-3dde-4768-a514-0f10aeed1790:0"
 #endif
 
+/* Maximum number of UEFI variables set to 100 */
+#define SMM_GATEWAY_MAX_UEFI_VARIABLES		(100)
+
 /* Default maximum number of UEFI variables */
 #ifndef SMM_GATEWAY_MAX_UEFI_VARIABLES
 #define SMM_GATEWAY_MAX_UEFI_VARIABLES		(40)
